scripts/docker_deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+version=$(grep -oP "PROJECT_VERSION \"?\K[0-9.]+(?=\")"? $(dirname "$0")/CMakeLists.txt)
+if [ "$TRAVIS_BRANCH" == "develop" ]; then
+	docker tag ethereum/solc:build ethereum/solc:nightly;
+	docker tag ethereum/solc:build ethereum/solc:nightly-"$version"-"$TRAVIS_COMMIT"
+	docker push ethereum/solc:nightly-"$version"-"$TRAVIS_COMMIT";
+	docker push ethereum/solc:nightly;
+elif [ "$TRAVIS_BRANCH" == "release"]; then
```

Comment:
It should not check for the branch, it should check for the `v...` tag on that branch.